joint_test_param_node:
  ros__parameters:
    input_in_degrees: true
    step_loop: false
    num_joints: 3
    delay_seconds: 4.0
    joint_sequence_flat:
      - 0.0
      - 45.0
      - 90.0
      - 0.0
      - 45.0
      - 60.0
